ci-operator/step-registry/aro-hcp/gather/snapshot/aro-hcp-gather-snapshot-commands.sh

```
@@ -0,0 +1,35 @@
+START_TIME_FALLBACK_ARGS=""
+if [[ -f "${SHARED_DIR}/write-config-timestamp-rfc3339" ]]; then
+  START_TIME_FALLBACK_ARGS="--start-time-fallback $(cat "${SHARED_DIR}/write-config-timestamp-rfc3339")"
+fi
+
+export AZURE_TOKEN_CREDENTIALS=prod
+test/aro-hcp-tests gather-snapshot \
+  --timing-input "${SHARED_DIR}" \
+  --output "${ARTIFACT_DIR}/" \
+  --rendered-config "${SHARED_DIR}/config.yaml" \
+  --subscription-id "${INFRA_SUBSCRIPTION_ID}" \
+  ${START_TIME_FALLBACK_ARGS}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)